addons/knowledge/static/src/components/permission_panel/permission_panel.js

```diff
@@ -0,0 +1,334 @@
+/** @odoo-module **/
+
+import session from 'web.session';
+import Dialog from 'web.Dialog';
+import { _t } from 'web.core';
+import { useService } from '@web/core/utils/hooks';
+
+const { Component, onWillStart, useState } = owl;
+const permissionLevel = {'none': 0, 'read': 1, 'write': 2}
+const restrictMessage = _t("Are you sure you want to restrict this role and restrict access ? "
```

Comment:
`_t` is an eager translation function => since the translations are not loaded when this file is loaded, they will not be applied. you should use `_lt` instead (lazy translate)